### DISCUSSION TITLE
Tarpaulin not coverage some file or partially

### DISCUSSION THREAD

**Discussion by julienvne** [xd009642/tarpaulin#1398] [Q&A]
When I run ```crago tarpaulin --all --out --html```, most of my files are ignored by the coverage. (it's a library)
In the report, they are marked 0/0 for the number of lines even though my files are not empty and I have tests for these same files.

Would you know how to fix this?

**Comment by xd009642** [comment]
So tarpaulin will filter out lines that can't be covered, and lines that are just in test code by default. And the line count is the number of "coverable lines". If it's filtering out code that can be ran then that's an issue. Do you have a repo I can look at?
